File: src/targets/java/okhttp/fixtures/multipart-form-data-no-params.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
<<<<<<< HEAD
  .url("https://httpbin.org/anything")
=======
  .url("http://mockbin.com/har")
>>>>>>> upstream/master
  .post(null)
  .addHeader("Content-Type", "multipart/form-data")
  .build();

Response response = client.newCall(request).execute();

File: src/targets/kotlin/okhttp/fixtures/multipart-form-data-no-params.kt
val client = OkHttpClient()

val request = Request.Builder()
<<<<<<< HEAD
  .url("https://httpbin.org/anything")
=======
  .url("http://mockbin.com/har")
>>>>>>> upstream/master
  .post(null)
  .addHeader("Content-Type", "multipart/form-data")
  .build()

val response = client.newCall(request).execute()